src/components/MultiSelect/pageObject/index.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * MultiSelect page object class.
+ * @class
+ */
+class PageMultiSelect {
+    /**
+     * Create a new MultiSelect page object.
+     * @constructor
+     * @param {string} rootElement - The selector of the MultiSelect root element.
+     */
+    constructor(rootElement) {
+        this.rootElement = rootElement;
+    }
+
+    /**
+     * Clicks the component
+     * @method
+     */
+    click() {
+        $(this.rootElement)
+            .$('div')
```

Comment:
I think we should be more specific in this selector like using the role or other attribute